Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspectorViewModel.js

```diff
@@ -325,11 +325,18 @@ define([
                 if (value) {
                     that._eventHandler.setInputAction(function(e) {
                         var picked = scene.pick(e.endPosition);
-                        if (picked instanceof Cesium3DTileFeature) {
+                        if (picked && defined(picked.content)) {
                             that.feature = picked;
+                            var position;
+                            if (scene.pickPositionSupported) {
+                                position = scene.pickPosition(e.endPosition);
+                                that._tileset._debugPickPosition = position;
+                            }
+                            that._tileset._debugPickedTile = picked.content.tile;
```

Comment:
This area could be organized a little differently so that it only does these steps if `showOnlyPickedTileDebugLabel` is true. Also setting `that.feature = picked` may have side effects when picked is not a feature.